lib/components/link.vue

```diff
@@ -13,8 +13,7 @@
        :event="event"
        :tag="tag"
        :class="linkClassObject"
-       @click="linkClick"
-    >
+       @click="linkClick">
```

Comment:
I wonder if this should be @click.native="linkClick($event)"`, as router link doesn't propagate a click event upwards.

See issue #534, and another few issues that I can't recall offhand)